### ISSUE TITLE
Possible to add export only dataview?

### ISSUE THREAD

**Issue by mithlug** [benjaminjackman/looty#20]
Maybe not an issue per se, but could we get an export dataview feature? In case I use your tool to narrow down and then export to some other program through .csv

Would save the sorting out all items in the .csv-file for everytime I add an item.

**Comment by benjaminjackman** [comment]
Good idea! I'll add that to the features to add list. It might be a while though as I don't have time to work on Looty in the near future.
